scripts/build-controllers-image.sh

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env bash
+
+set -E
+
+DIR=$(cd "$(dirname "$0")"; pwd)
+SCRIPTS_DIR=$DIR
+DOCKERFILES_DIR=$DIR/../docker
+BUILD_CONTEXT=$DIR/..
+
+: "${DOCKERFILE:="$DOCKERFILES_DIR/Dockerfile"}"
+
+source $SCRIPTS_DIR/lib/common.sh
+source $SCRIPTS_DIR/lib/docker.sh
+
+check_is_installed docker
+ensure_docker_daemon_running || exit 1
+
+USAGE="
+Usage:
+  $(basename "$0") <services>
+
+<services> should be a space-delimited list of AWS service API aliases
+that you wish to build container iamges for -- e.g. 's3 sns sqs'
+"
+
+SERVICES=()
+
+while [ -n "$1" ]; do
+    SERVICES+=( $1 )
+    shift
+done
+
+if [[ ${#SERVICES[@]} -eq 0 ]]; then
+    echo "ERROR: Specify at least one service to build an container image for" 1>&2
+    echo "$USAGE"
+    exit 1
+fi
+
+for SERVICE in ${SERVICES[@]}; do
+    IMAGE_TAG=ack/$SERVICE
```

Comment:
IMAGE_TAG should have a tag suffix (otherwise it defaults to ":latest" which isn't a best practice. Can you default the tag to the git tag or dirty, like the kind-build-test.sh script does:

```
VERSION=$(git describe --tags --always --dirty || echo "unknown")
DEFAULT_AWS_SERVICE_DOCKER_IMG="ack-${AWS_SERVICE}-controller:${VERSION}"
```